echopype/convert/convert.py

```diff
@@ -35,6 +35,26 @@
 NMEA_SENTENCE_DEFAULT = ["GGA", "GLL", "RMC"]
 
 
+def open_raw(file=None, model=None, xml_path=None, storage_options=None):
+    """Create a converter object by reading in raw data files.
+
+    The converter object can be used for adding metadata and ancillary data
```

Comment:
Maybe it'd be a bit more helpful to say "Create a Convert object ...", "The Convert object ..."? But this is a minor point.

Reply:
Agreed, better to be explicit